src/pages/workspaces/workspace/Dashboard/Dashboard.ts

```diff
@@ -313,7 +314,7 @@ const WorkspaceDashboardForwardRefRenderFunction = (
               style: { marginLeft: '0.5rem' },
               disabled: !canEdit,
               tooltip: canEdit ? 'Edit description' : editErrorMessage,
-              onClick: () => setEditDescription(description?.toString()),
+              onClick: () => setEditDescription(description),
```

Comment:
This was causing a problem with how `isEditing` is determined at line 301, as `isEditing` would end up being false if `description` was undefined.

```
const isEditing = _.isString(editDescription);

Reply:
Wouldn't it still be false if description is undefined? It looks exactly the same to me, just without converting a non-string value to a string now.

Reply:
No, because of https://github.com/DataBiosphere/terra-ui/pull/4502/files#diff-c1a6f75cfb1d5746258886b8efb91ec8a6a7052b43662bb96aa0db5d8acae4aaR122.

Reply:
It will now be an empty string instead of undefined.